_docs/examples/provider/README.md

```diff
@@ -153,17 +166,24 @@ EOF
 
 ### Export keys
 
+**Replace "password" below with your own unique secret.**
+
 ```sh
 echo "password" > key-pass.txt
-(cat key-pass.txt; cat key-pass.txt) | ./bin/akashctl keys export provider 2> key.txt
+(cat key-pass.txt; cat key-pass.txt) | akash --home=$AKASH_HOME keys export provider1 2> key.txt
 ```
 
 ### Configure Kubernetes Akash Namespace
 
 Create the `akash-services` namespace for running Provider.
 
 ```sh
-kubectl apply -f https://raw.githubusercontent.com/ovrclk/akash/master/_docs/kustomize/namespace.yaml
+kubectl apply -f https://raw.githubusercontent.com/ovrclk/akash/master/_docs/kustomize/networking/namespace.yaml
```

Comment:
I tend to agree with your point, however this `namespace.yaml` might be one of the few exceptions since it probably won't ever change. 

For example files like `deployment.yaml`, or `kustomization.yaml` which will have version specific changes, are more fragile to break referencing master.